Resources/Prototypes/_Sunrise/Catalog/Fills/Crates/ussp.yml

```
@@ -0,0 +1,223 @@
+            max: 3
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
+
+# Упакованная Винтовка для ящика мосинок
+- type: entity
+  parent: [BaseItem, BaseSecurityContraband]
+  id: WrappedMosin
+  name: wraped mosin rifle
+  description: A Kardashev-Mosin relic rifle, created 670 years ago. Hermetically sealed for eternal storage — heavily greased with cosmic lubricant, tightly wrapped in packing paper, and forgotten until the next war. Ammunition not included.
+  components:
+  - type: Item
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)